client/sites/tools/staging-site/components/staging-site-card/index.js

```diff
@@ -415,8 +417,16 @@ export const StagingSiteCard = ( {
 	} );
 
 	const { pullFromStaging } = usePullFromStagingMutation( siteId, stagingSite?.id, {
-		onSuccess: () => {
-			dispatch( recordTracksEvent( 'calypso_hosting_configuration_staging_site_pull_success' ) );
+		onSuccess: ( data, variables ) => {
+			dispatch(
+				recordTracksEvent(
+					'calypso_hosting_configuration_staging_site_pull_success',
+					zipObject(
```

Comment:
How could we make it clearer and easier to understand? Would it make sense to rename `variables` to `sync_options` or similar? Should we prefix them somehow to organize them in Tracks app?